### ISSUE TITLE
Trojan:AndroidOS/Multiverze False positive

### ISSUE THREAD

**Issue by maybedisp** [hannesmann/vanillafixes#22]
Windows defender is detecting version 1.3 as Trojan:AndroidOS/Multiverze false positive, but the old version works without antivirus freaking out. This seems to be a repeat of #17

**Comment by hannesmann** [comment]
I tried scanning all three ZIP files but I still can't reproduce the detection. Not sure what could be done to fix this :/
https://i.imgur.com/IvyVrQR.png

**Comment by maybedisp** [comment]
Strange, I even tried removing clearing AV definitions/dynamic signatures and rescanned the file and it still gets detected.
![vanillafixes 1 3](https://github.com/hannesmann/vanillafixes/assets/114700006/c9a5ed1d-11dd-4df0-84a0-59ce03aae80d)

**Comment by hannesmann** [comment]
I figured out why it wasn't detected for me, I had "Cloud-delivered protection" turned off. vanillafixes-1.3.zip is detected as "Trojan:AndroidOS/Multiverze" and vanillafixes-1.3-dxvk-legacy.zip is detected as "Trojan:Script/Wacatac.H!ml" but vanillafixes-1.3-dxvk.zip is not detected. 

I couldn't get VanillaFixes.exe to trigger any detection and Android apps are ZIP files which leads me to believe that Defender is detecting the ZIP file *itself* as a virus and not the contents (which is really stupid). 

I submitted vanillafixes-1.3.zip to Microsoft and removed vanillafixes-1.3-dxvk-legacy.zip. They responded pretty fast in #17 so hopefully the false positive will be removed soon.
